client/lib/cart-values/cart-items.js

```diff
@@ -802,7 +804,7 @@ module.exports = {
 	hasDomainRegistration,
 	hasFreeTrial,
 	hasGoogleApps,
-	hasNlTld,
+	hasTld,
```

Comment:
This list should be kept in alphabetical order, so now this function has to be moved a bit lower :)